tests/std/tests/P2474R2_views_repeat_death/test.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _CONTAINER_DEBUG_LEVEL 1
+
+#include <ranges>
+
+#include <test_death.hpp>
+
+using namespace std;
+using ranges::repeat_view;
+
+void test_view_negative() {
+    [[maybe_unused]] auto v = repeat_view(1, -1); // bound must be positive

```

Comment:
This constructor is overloaded on `const T&` and `T&&`, so we should test both, as they have separate checks.